index.html

```diff
@@ -58,5 +64,142 @@ <h1>Hey there. I’m <br>Lukman
+                    <div class="projectdetail">
+                        <h2 class="projectname">Project 1</h2>
+                    <ul class="skills">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button type="button" class="projectbutton">See Project</button>
+                </div></div>
+                <div class="project2"></div>
+            </div>    
+            
+        </div>
+        </main>
+    </section>
+    <section>
+        <div class="container-about">
+            <div class="aboutimage"><h2>About me <img class="img-foot" src="/images/illustration 1- about me-mobile.svg" alt=""> </h2></div>
+            <p> I can help you build a product , feature or website
+                 Look through some of my work and experience!
+                  If you like what you see and have a project
+                   you need coded, 
+                <br>don’t hestiate to contact me.</p>
+            <button type="button" class="projectbutton">Get My Resume</button>
+           
+            <div class="projectlist about">
+                <div class="project1 about">
+                    <div class="image about">
+                        <img class="projectimage about" src="/images/icon-languages.svg" alt="project desc"></div>
+                    <div class="projectdetail about">
+                        <h2 class="projectname about">Languages</h2>
+                    <ul class="skills about">
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                </div></div>
+            </div>
+            <div class="projectlist about">
+                <div class="project1 about">
+                    <div class="image about">
+                        <img class="projectimage about" src="/images/icon-frameworks.svg" alt="project desc"></div>
+                    <div class="projectdetail about">
+                        <h2 class="projectname about">Frameworks</h2>
+                    <ul class="skills about">
+                        <li>Bootstrap</li>
+
+                    </ul>
+                </div></div>
+            </div>
+            <div class="projectlist about">
+                <div class="project1 about">
+                    <div class="image about">
+                        <img class="projectimage about" src="/images/icon-skills.svg" alt="project desc"></div>
+                    
+                        <h2 class="projectname about">Skills</h2>
+                    <ul class="skills about">
+                        <li>Github</li>
+                        <li>Terminal</li>
+                        <li>Codepen</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
```

Comment:
Kindly note that _The technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, each of them in a `li`. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") should also be a list `ul` and each of them a `li`._
Your technologies are in a list but your categories are not, so kindly fix that 🤗

Reply:
Hello, I thank you for your generous comments. 
I want to clarify, do you mean something like this? 
<ul class="projectlist about">
                <div class="project1 about">
                    <div class="image about">
                        <img class="projectimage about" src="/images/icon-languages.svg" alt="project desc"></div>
                        <li><h2 class="projectname about">Languages</h2></li>
                    <li><ul class="skills about">
                        <li>CSS</li>
                        <li>JavaScript</li>
                        <li>HTML</li>
                    </ul></li>
                
            </ul>

Reply:
>  <ul class="projectlist about">
                <div class="project1 about">
                    <div class="image about">
                        <img class="projectimage about" src="/images/icon-languages.svg" alt="project desc"></div>
                        <li><h2 class="projectname about">Languages</h2></li>
                    <li><ul class="skills about">
                        <li>CSS</li>
                        <li>JavaScript</li>
                        <li>HTML</li>
                    </ul></li>
                
            </ul>

Reply:
@awais-amjed

Reply:
Hi, really sorry for not responding. I didn't get any notification about the comment even though I have turned them on, I will check my notifications again though. But anyway you did a great job on implementing the requested changes 👏👏